lib/models/repository-states/present.js

```diff
@@ -466,16 +533,14 @@ export default class Present extends State {
     });
   }
 
-  getAheadCount(branchName) {
-    return this.cache.getOrSet(Keys.aheadCount.oneWith(branchName), () => {
-      return this.git().getAheadCount(branchName);
-    });
+  async getAheadCount(branchName) {
+    const bundle = await this.getStatusBundle();
+    return bundle.branch.aheadBehind ? bundle.branch.aheadBehind.ahead : null;
```

Comment:
This might be cleaner if `bundle.branch.aheadBehind` was always present and just defaulted to `{ahead: null, behind: null}`. Unless there's a case where `assert.isNull(aheadBehind)` and `assert.deepEqual(aheadBehind, {ahead: null, behind: null}` are both valid and mean different things 🤔